src/DataTable/README.md

```diff
@@ -253,7 +253,7 @@ To enable proper selection behavior with backend pagination (i.e., when ``isSele
     </Component>
   );
 
-  const ClearAction = ({ as: Component, tableInstance, ...rest }) => (
+  const ClearAction = ({ as: Component, tableInstance }) => (
```

Comment:
The `rest` parameter was unused.

Fixes: `'rest' is declared but its value is never read.`